home.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ] [ OPTIONAL ] Please consider linking your style.css file here so that your CSS file can be correctly applied to your HTML tags. Here is a possible suggestion: `<link rel="stylesheet" href="style.css">` 👍